src/app/Routes.js

```diff
@@ -148,6 +149,16 @@ const Routes = () => {
           }}
         />
       ) : null}
+      {user.data && user.data.type === 'RESEARCHER' ? ( // Se o usuário for um pesquisador, ele pode adicionar outros pesquisadores
+        <Stack.Screen
+          name="AddContributor"
+          component={AddContributor} // chamando pagina de inclusão de usuário (com o ctr + click tu pode ser direcionado a ela)
```

Comment:
```suggestion
          component={AddContributor}
```
Também acho que não precisa do comentário no código, é interessante esse tipo de comentários em anotações internas, pq meio que o código já é auto explicável.